src/pipecat/pipeline/task.py

```diff
@@ -810,7 +826,7 @@ async def _idle_timeout_detected(self) -> bool:
         """
         # If we are cancelling, just exit the task.
         if self._cancelled:
-            return True
+            return False
```

Comment:
What effect does flipping this value have?

Reply:
Ah right `False` means stop. OK I got turned around. This makes sense. No need to keep running the timeout checker. 👍